tests/unit/test_admin_pipeline.py

```
@@ -0,0 +1,176 @@
+                
+            elif response.status_code == 429:
+                print(f"⚠️  Rate Limited (expected for some scenarios)")
+                error_data = response.json()
+                print(f"   Error: {error_data.get('error', 'Unknown')}")
+                
+            else:
+                print(f"❌ Failed!")
+                try:
+                    error_data = response.json()
+                    print(f"   Error: {error_data.get('error', 'Unknown')}")
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)